src/components/Items/MyItemListings.jsx

```diff
@@ -1,64 +1,110 @@
 import GridItem from 'components/MaterialKitComponents/Grid/GridItem';
-import MyItemListing from 'components/Items/MyItemListing';
+import MyItemListing from 'containers/Items/MyItemListing';
 import ListingsDropdown from 'components/Items/ListingsDropdown';
 import Button from 'components/CustomButtons/Button';
 import ListingsPaginations from 'components/Items/ListingsPaginations';
 import PropTypes from 'prop-types';
 
 export default function MyItemListings(props) {
-  const { type, items } = props;
+  console.log('*** MY ITEM LISTINGSSSS PROPS:', props);
+  const { _id, type, myitems } = props;
+
+  // console.log('*** items:', props.items);
+  // useEffect(() => {
+  //  console.log('Calling MyItemListings UseEffect');
+  //  getProviderItems(_id);
+  // }, []);
+  // useEffect(() => {
+  //   getProviderItems(_id);
+  //   getCollectorItems(_id);
+  // }, [getProviderItems, getCollectorItems]);
+
   const classes = makeStyles(styles);
   const paginationColSize = type === 'provide' ? 6 : 12;
 
   const sortItems = menuItem => {
     props.sortByItem(menuItem);
   };
 
-  const deleteItem = async itemId => {
-    // console.log(`Clicked Delete button, delete item with itemId: ${itemId}`);
-    await props.deleteItem(itemId);
+  const deleteItem = async _id => {
+    // console.log(`Clicked Delete button, delete item with _id: ${_id}`);
+    await props.deleteItem(_id);
   };
 
+  const unreserveItem = async (_id, itemId) => {
+    // console.log(`Clicked Delete button for collector, unreseve item with _id: ${_id}`);
+    await props.unreserveItem(_id, itemId);
+  };
+
+  const getMyItems = () =>
+      // eslint-disable-next-line implicit-arrow-linebreak
+     myitems.filter(myItem => {
+      if (type === 'provide') {
+        if (myItem.user_uid === props._id) {
+          return myItem;
+        }
+      } else if (myItem.c_user_uid === props._id) {
+        return myItem;
+      }
+      return null;
+  });
+
   return (
-    <div>
-      <GridContainer>
-        <GridItem align="right">
-          <ListingsDropdown sortItems={sortItems} />
-        </GridItem>
-        <GridItem xs={12} sm={12} md={12}>
-          {items
-            .filter(item => props.uuid === (type === 'provide' ? item.provideUserId : item.collectUserId))
-            .map(myItem => (<MyItemListing key={myItem.itemId} type={type} myitem={myItem} deleteItem={deleteItem} />))}
-        </GridItem>
-        {type === 'provide' && (
-          <GridItem xs={6} sm={6} align="left">
-            <Link to="/addedititem" className={classes.link}>
-              <Button type="button" color="rose" to="/addedititem">
-                Add Item
-              </Button>
-            </Link>
-          </GridItem>
-        )}
-        <GridItem xs={paginationColSize} sm={paginationColSize} align="right">
-          <ListingsPaginations />
-        </GridItem>
-      </GridContainer>
-    </div>
+    <>
+      {1==2 ? (
```

Comment:
Not sure what's going on here, is this a placeholder? Could you could just put false?